argus/backend/service/notification_manager.py

```diff
@@ -11,19 +13,53 @@ class NotificationManagerException(Exception):
 
 
 class NotificationManagerService:
+
+    def __init__(self) -> None:
+        self.notification_services = []
+        self.attach_notification_sender(ArgusDBNotificationSaver())
+        self.attach_notification_sender(EmailNotificationServiceSender())
+
+    def attach_notification_sender(self, service):
+        self.notification_services.append(service)
+
+    def send_notification(self, receiver: UUID, sender: UUID,
+                          notification_type: ArgusNotificationTypes,
+                          source_type: ArgusNotificationSourceTypes,
+                          source_id: UUID, title: str | None = None,
+                          content: str | None = None, content_params: dict | None = None) -> Any:
+
+        for service in self.notification_services:
+            service.send_notification(receiver, sender, notification_type, source_type,
+                                      source_id, title, content, content_params)
+
+    def get_notificaton(self, receiver: UUID, notification_id: UUID) -> ArgusNotification:
+        return ArgusNotification.get(receiver=receiver, id=notification_id)
+
+    def get_unread_count(self, receiver: UUID) -> int:
+        query = ArgusNotification.filter(
+            receiver=receiver, state__eq=ArgusNotificationState.UNREAD.value).allow_filtering().all()
+        return len(query)
+
+    def read_notification(self, receiver: UUID, notification_id: UUID) -> ArgusNotification:
```

Comment:
maybe rename it to `mark_notification_as_read`?

Reply:
@k0machi WDYT?

Reply:
I like `verb_noun` scheme more, but sure, that can be changed

Reply:
Maybe it is just me, but current name was not clear enough. E.g. read_notification could be understood as fetching it from db and showing to user, while this method just marks notification as read.

Reply:
@soyacz this method was not changed by this commit. Better to rename it in another one